src/models/userHealthInfoDao.js

```diff
@@ -0,0 +1,237 @@
+const { AppDataSource } = require("./dataSource");
+
+const checkExistence = async (userId) => {
+  const exist = await AppDataSource.query(
+  ` SELECT id FROM users where id = ?`,
+  [userId]
+  );
+  return exist.length === 0 ? false : true;
+  };
+
```

Comment:
```suggestion

const checkExistence = async (userId) => {
  const exist = await AppDataSource.query(
  ` SELECT id FROM users where id = ?`,
  [userId]
  );
  return exist.length === 1;
  };

```
사용자는 1명 뿐이어야 하니, length 가 1 인지 여부를 곧바로 전달 하면, 불필요한 3항 연산자를 줄일 수 있을 것 같습니다 :)

Reply:
와 그렇네요!
어차피 `exist.length === 1`  조건에서 true/false 값이 전달이 되네요
크으! 이마를 탁 칩니다!

Reply:
리뷰반영 커밋에 적용하였습니다.